streaming/src/main/scala/org/apache/spark/streaming/StreamingContext.scala

```diff
@@ -662,6 +664,16 @@ object StreamingContext extends Logging {
     }
   }
 
+  /**
+   * Get the context that is currently active, that is, started but not stopped.
```

Comment:
Nit, but I think maybe "Get the currently active context, if there is one. Active means started but not stopped." I.e. currently the doc sort of implies there is always a currently active context.